### PR TITLE
Gameroom walkthrough organization updates

### PR DESCRIPTION
**Pull request by vaporos**

### CODE REVIEW

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+
+![](./images/acme_gameroom_login_screen.svg "Gameroom app Acme login screen")
+
+Alice enters her EMAIL and PASSWORD. Clicks LOG IN.
+
+Success. The browser now displays the ACME GAMEROOM HOME SCREEN.
+
+![](./images/acme_gameroom_home_screen.svg "Gameroom app Acme home screen")
+
+## Behind the Scenes: A Look at Act I, Alice and BobCreate a Gameroom
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:23]: Typo: Missing a space (BobCreate)

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 2]
```diff
@@ -1,4 +1,28 @@
-## I-1. Behind scene 1: Alice logs into Acme's Gameroom UI
+
+# Gameroom Technical Walkthrough
+
+***NOTE:*** This document is a work-in-progress! We are updating the
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:4]: Ungrammatical: No hyphens should be used in "work in progress"

File: docs/0.4/examples/gameroom/walkthrough/index.md [hunk 3]
```diff
@@ -1,4 +1,28 @@
...
+walkthrough with details for Splinter 0.4 and converting it to markdown. For
```
Comment (chenette) [inline docs/0.4/examples/gameroom/walkthrough/index.md:5]: Not standard: Markdown is written with a capital M.

File: docs/0.4/examples/gameroom/walkthrough/index.md (hunk 1 above)
Comment (vaporos) [reply docs/0.4/examples/gameroom/walkthrough/index.md]: fixed

File: docs/0.4/examples/gameroom/walkthrough/index.md (hunk 2 above)
Comment (vaporos) [reply docs/0.4/examples/gameroom/walkthrough/index.md]: fixed

File: docs/0.4/examples/gameroom/walkthrough/index.md (hunk 3 above)
Comment (vaporos) [reply docs/0.4/examples/gameroom/walkthrough/index.md]: fixed

**Pull request merged**